README.md

```diff
@@ -291,65 +290,70 @@ and it is far easier to work with them when you have deeper understanding yourse
+  Equity compensation in each of these kinds of business entities is significantly different
+  than in C corporations.
+- 🄳 A [**C corporation (or C corp)**](https://en.wikipedia.org/wiki/C_corporation) is one type of
+  corporation United States.
   Specifically, it’s a kind of
-  [**stock corporation**](https://en.wikipedia.org/wiki/Joint-stock_company) with certain federal
-  tax treatment.
-  Most large companies you’ve heard of, as well as startups that aspire to be large, are C
-  corporations. In practice, for a
+  [**stock corporation (or joint-stock company)**](https://en.wikipedia.org/wiki/Joint-stock_company))
+  with certain federal tax treatment.
+  Many of the high-profile companies you hear about are C corporations.
+- In practice, for a
   [few](https://www.nytimes.com/2012/07/01/business/how-delaware-thrives-as-a-corporate-tax-haven.html)
   [reasons](https://www.quora.com/Why-do-most-technology-startups-incorporate-in-Delaware),
   these companies are usually formed in Delaware, so legalities of all this are defined in
   Delaware law.
-- Loosely, one way to think about all this is that a company is a set of contracts,
-  negotiated over time between people and enforced by the government, that align interests
-  of a group of people in creating things customers are willing to pay for.
-  A key part of these contacts is precisely tracking ownership.
-  Historically, this has been done a few ways, depending on the size and nature of the
-  company, but possibly the most effective way when a lot of people are involved, is via
-  stock, which we get into next.
+
+Some of what we discuss is specific to early stage companies.
+
+- 🄳 A [**startup**](https://en.wikipedia.org/wiki/Startup_company) is an emerging company,
+  typically a [private company](#public-and-private-companies), that aspires to grow quickly in
+  size, revenue, and influence.
+  Once a company is established in the market and successful for a while, it usually stops
+  being called a startup.
+- ☝️ Unlike the terminology around corporations, which has legal significance, the term
+  *startup* is informal, and not everyone uses it consistently.
+  Distinguished investor [Paul Graham](https://en.wikipedia.org/wiki/Paul_Graham_(programmer))
+  [emphasizes](http://www.paulgraham.com/growth.html) suggests that it’s best to think of a
+  startup as any [early stage](#stages-of-a-startup) company intending to grow quickly.
+
+Loosely, one way to think about this is that companies are simply a set of contracts,
+enforced by the government and negotiated over time between the people owning and
+operating it, that aligns the interests of everyone involved in creating things customers
+are willing to pay for.
+A key part of these contacts is precisely tracking ownership.
+Historically, this has been done a few ways, depending on the size and nature of the
+company, but possibly the most effective way when a lot of people are involved, is via
+stock, which we get into next.
```

Comment:
This sentence yields more questions than answers. In #114 I move this to the bottom of Public and Private Companies so that the transition b/n info on companies and the next section on Stock and Shares is clear (currently, following Stock and shares with more info on types of companies is very confusing, distracts from the significance of this transitioning material), and shortened it for clarity: 
"+- Loosely, one way to think about all this is that a company is a set of contracts,
 +  negotiated over time between people and enforced by the government, that aligns the
 +  interests of a group of people in creating things customers are willing to pay for.
 +  Key to these contracts is a way to precisely track ownership of a company;
 +  the issuing of stock is one way companies can choose to do this."

Reply:
Like the last sentence, will add.